tensorflow/examples/tutorials/mnist/mnist_softmax_xla.py

```diff
@@ -102,6 +102,14 @@ def main(_):
       default='/tmp/tensorflow/mnist/input_data',
       help='Directory for storing input data')
   parser.add_argument(
-      '--xla', type=bool, default=True, help='Turn xla via JIT on')
+      '--no-xla',
+      dest='xla', action='store_false',
```

Comment:
I've preferred this alternative version of specifying boolean flags in argparse:

https://github.com/tensorflow/tensorflow/blob/ec7929b878926c39255254e9aea992f0bc65aa68/tensorflow/python/debug/examples/debug_errors.py#L72

This allows you to do --xla=True or --xla and have it work.  What do you think?